README.md

```diff
@@ -90,7 +90,7 @@ GitHub issues, but instead follow
 guidelines](https://github.com/firecracker-microvm/firecracker/blob/master/SECURITY-POLICY.md).
 
 Other discussion: For general discussion, please join us in the `#containerd`
-channel on the [Firecracker Slack](https://tinyurl.com/firecracker-microvm).
+channel on the [Firecracker Slack](https://join.slack.com/t/firecracker-microvm/shared_invite/zt-oxbm7tqt-GLlze9zZ7sdRSDY6OnXXHg).
```

Comment:
Any reason why we are not using tinyurl anymore?

Reply:
Yeah, it can't re-generate the `https://tinyurl.com/firecracker-microvm` with the original url. If we decide to use tinyurl, we would have something like `https://tinyurl.com/[some_random_characters]`. Both Kazu and I agree to replace the tinyurl with the original url. Firecracker team use the original url as well. Besides, the firecracker-go-sdk also uses the original url.